### PR TITLE
Remove COBALT_LINUX macro

### PR DESCRIPTION
**Pull request by kaidokert**
Unused, except for a small sandbox exclusion.

b/150410605

### CODE REVIEW

File: starboard/linux/x64x11/gcc/6.3/starboard_loader.cc [hunk 1]
```diff
@@ -13,8 +13,20 @@
...
+namespace std {
+// This refers to a macro that should normally resolve to __1, __2 or be
+// defined by LIBCXX_ABI_NAMESPACE or LIBCXX_ABI_VERSION
+namespace _LIBCPP_ABI_NAMESPACE {
+__attribute__((visibility("default"))) std::istream& cin = ::std::cin;
+}
+}  // namespace std
+
 int main(int argc, char** argv) {
   printf("starboard_loader: SbRunStarboardMain\n");
```
Comment (andrewsavage1) [inline starboard/linux/x64x11/gcc/6.3/starboard_loader.cc:31]: Not your change ofc, but this seems like an unnecessary printf that can be removed

**Review by andrewsavage1 — APPROVED** [review]

File: starboard/linux/x64x11/clang/3.9/starboard_loader.cc [hunk 2]
```diff
@@ -12,8 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <iostream>
+
 #include "starboard/event.h"
 
+// Provide a workaround alias for newer libcxx that places iostreams in
+// a namespace.
+namespace std {
```
[the remark this answers is not in the document]
Comment (kaidokert) [reply starboard/linux/x64x11/clang/3.9/starboard_loader.cc:21]: I'll give it a go, trying to find a reasonable place to put it. I put it here as first pass as this is the most brute force and `always works` method to get linkers to include the symbols.

File: starboard/linux/x64x11/gcc/6.3/starboard_loader.cc (hunk 1 above)
Comment (kaidokert) [reply starboard/linux/x64x11/gcc/6.3/starboard_loader.cc]: Cleaned that up.

File: starboard/linux/x64x11/clang/3.9/starboard_loader.cc (hunk 2 above)
[the remark this answers is not in the document]
Comment (kaidokert) [reply starboard/linux/x64x11/clang/3.9/starboard_loader.cc]: Seems to have worked.

**Pull request merged**